### ISSUE TITLE
Resizing the window breaks ofGetWidth & ofGetHeight

### ISSUE THREAD

**Issue by braitsch** [openframeworks/openFrameworks#4842]
Hi, it looks like ofGetWidth is acting up again. If I enable hi-res ofGetWidth reports 2x the window width on window resize but only reports 1x everywhere else. To see what I mean create an empty project, enable hi-res and print out the width within setup, a keypress and resize handler.

````
void ofApp::setup(){
    cout << "ofGetWidth() " << endl; // 1024
}
void ofApp::keyReleased(int key){
    cout << "ofGetWidth() " << endl; // 1024
}
void ofApp::windowResized(int w, int h){
    cout << "ofGetWidth() " << endl; // 2048
}
````
OF v0.9.2 
OSX v10.10.5

**Comment by braitsch** [comment]
I found the problem. The pixelScreenCoordScale multiplication was happening twice, once in the resize callback and again in the getWidth call. 
````
void ofAppGLFWWindow::resize_cb(GLFWwindow* windowP_,int w, int h) {
	ofAppGLFWWindow * instance = setCurrent(windowP_);
	cout << "w = " << w << " scale = " << instance->pixelScreenCoordScale << endl;
// don't modify instance dimensions here since queries (e.g. getWidth & getHeight)  
// will return the window values adjusted by the pixelScreenCoordScale
	instance->windowW = w;// * instance->pixelScreenCoordScale;
	instance->windowH = h;// * instance->pixelScreenCoordScale;
	cout << "sending " << w*instance->pixelScreenCoordScale << endl;
	instance->events().notifyWindowResized(w*instance->pixelScreenCoordScale, h*instance->pixelScreenCoordScale);
	instance->nFramesSinceWindowResized = 0;
}

int ofAppGLFWWindow::getWidth(){
	if(windowMode == OF_GAME_MODE)
	{
		return getScreenSize().x;
	}
	else {
		if( orientation == OF_ORIENTATION_DEFAULT || orientation == OF_ORIENTATION_180 ){
			return windowW * pixelScreenCoordScale;
		}else{
			return windowH * pixelScreenCoordScale;
		}
	}

}
````

I'll submit a pull request in a few minutes.

**Comment by braitsch** [comment]
Noting that this bug was introduced in v0.9.1 and breaks any app running in hi-res fullscreen mode that relies on querying the window dimensions to lay itself out.

**Comment by arturoc** [comment]
just milestoned this for 0.9.3. @ofTheo can you take a look? Also @braitsch can you send this PR against the 0.9.3 branch?

**Comment by danoli3** [comment]
S%I@(#$, yeah I'll fix that

**Comment by ofTheo** [comment]
Will do!!

**Comment by braitsch** [comment]
@arturoc [just submitted a PR against the 0.9.3 branch](https://github.com/openframeworks/openFrameworks/pull/4858). thanks fellas!

**Comment by ofTheo** [comment]
closed by #4858
